applet/src/console.c

```diff
@@ -70,7 +64,7 @@ static void con_addchar( char c )
 {
     // intentional non-wrap.
     if( con_xpos >= MAX_XPOS ) {
-        return ;
+        con_nl();
```

Comment:
and please study syslog.c some more, the distinction between wrapping in console en wrapping in syslog is actually there.